From a0ad15de26d196efc67c131959bad5773c02a719 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tqs.de>
Date: Mon, 10 Sep 2018 11:12:02 +0200
Subject: [PATCH 004/188] DT: Vendor: add prefix for TQ Systems

Add prefix for TQ Systems GmbH and for TQ Components
both are needed since the existing PPC device trees use
tqc but the TQMa53 ARM module uses tq. The tq prefix is better
since TQ Components was renamed some time ago.

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 Documentation/devicetree/bindings/vendor-prefixes.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/vendor-prefixes.txt b/Documentation/devicetree/bindings/vendor-prefixes.txt
index d8eac2dcfaa7..f04f1a8e595f 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.txt
+++ b/Documentation/devicetree/bindings/vendor-prefixes.txt
@@ -349,6 +349,8 @@ toumaz	Toumaz
 tpk	TPK U.S.A. LLC
 tplink	TP-LINK Technologies Co., Ltd.
 tpo	TPO
+tq	TQ Systems GmbH
+tqc	TQ Components (deprecated, use tq)
 tronfy	Tronfy
 tronsmart	Tronsmart
 truly	Truly Semiconductors Limited
